sources/logs.datatables.php

```diff
@@ -214,10 +213,15 @@
     $sSearch = $params['sSearch'] ?? '';
     if ($sSearch !== '') {
         $sWhere .= ' AND (';
+        $sanitizedSearch = filter_var($sSearch, FILTER_SANITIZE_FULL_SPECIAL_CHARS);
```

Comment:
This does not sufficiently protect us against SQL injection.

Reply:
Should be better now ;)